### PR TITLE
qa/tests - added luminous-p2p suite to the schedule

### PR DESCRIPTION
**Pull request by yuriw**
(Merge after https://github.com/ceph/ceph/pull/22474 is merged)

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -105,9 +105,11 @@ CEPH_QA_EMAIL="ceph-qa@ceph.com"
...
+45 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 2 -m $MACHINE_NAME -s upgrade/client-upgrade-hammer -k distro -e $CEPH_QA_EMAIL --suite-branch hammer
+47 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 2 -m $MACHINE_NAME -s upgrade/client-upgrade-jewel -k distro -e $CEPH_QA_EMAIL --suite-branch jewel --filter "ubuntu_14.04,ubuntu_16.04,centos_7.4"
+50 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=ovh;   /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 2 -m $MACHINE_NAME -s upgrade/client-upgrade-kraken -k distro -e $CEPH_QA_EMAIL --suite-branch kraken
+## point-to-point upgrades suites on luminous
+30 05 * * 7  CEPH_BRANCH=luminous; MACHINE_NAME=smithi;/home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 2 -m $MACHINE_NAME -s upgrade/luminous-p2p -k distro -e $CEPH_QA_EMAIL
```
Comment (neha-ojha) [inline qa/crontab/teuthology-cronjobs]: doesn't it need a --suite-branch?

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: no it will use `luminous`

**Comment by neha-ojha** [comment]
nit: typo in commit message "siute"

**Comment by yuriw** [comment]
fixed in commit

**Pull request merged**